Subject: gpio: GPIO framework
From: Robert Schwebel <r.schwebel@pengutronix.de>

This patch adds ARM support to generic GPIO framework.

Not intended for mainline.


Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
Signed-off-by: Juergen Beisert <j.beisert@pengutronix.de>

--- 

 arch/arm/Kconfig |    2 ++
 1 file changed, 2 insertions(+)

Index: arch/arm/Kconfig
===================================================================
--- arch/arm/Kconfig.orig
+++ arch/arm/Kconfig
@@ -1048,6 +1048,8 @@ source "drivers/rtc/Kconfig"
 
 source "drivers/dma/Kconfig"
 
+source "drivers/gpio/Kconfig"
+
 endmenu
 
 source "fs/Kconfig"
Index: include/asm-arm/gpio.h
===================================================================
--- include/asm-arm/gpio.h.orig
+++ include/asm-arm/gpio.h
@@ -4,4 +4,8 @@
 /* not all ARM platforms necessarily support this API ... */
 #include <asm/arch/gpio.h>
 
+/* policies for the userland part */
+#define GPIO_USER       (1<<0)
+#define GPIO_OUTPUT     (1<<1)
+
 #endif /* _ARCH_ARM_GPIO_H */
